sdk/sdktypes/trigger.go

```diff
@@ -19,10 +19,11 @@ type TriggerPB = triggerv1.Trigger
 type TriggerTraits struct{}
 
 func (TriggerTraits) Validate(m *TriggerPB) error {
+	// FIXME: where this check belongs? Validate or StrictValidate?
```

Comment:
`StrictValidate`. In `Validate` it just check each field individually. In `Strict` it checks the combination.

Reply:
hmm.. just trying to clarify for myself what you wrote - according to the code both `Validate` and `strictValidate` check each field individually and return combination of errors, no? The only difference that StrictValidate won't allow empty fields

I actually think that this check should be in `Validate`. The natural place of this check is in `proto/buf` code. But we cannot check it unless we defined either schedule as a field or `data` as a structure. So, by replacing `buf/proto` check by `sdktype` check we are trying to achieve the same coverage, therefore IMO the logical place is in `Validate` function to mimic the unexising check from `buf/proto`, no?